packages/breadcrumbs/src/styled/StyledChevronIcon.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useContext, SVGAttributes } from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import ChevronLeftStrokeIcon from '@zendeskgarden/svg-icons/src/12/chevron-left-stroke.svg';
+import ChevronRightStrokeIcon from '@zendeskgarden/svg-icons/src/12/chevron-right-stroke.svg';
+import { DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const LTR_COMPONENT_ID = 'breadcrumbs.chevron_right_stroke_icon';
+const RTL_COMPONENT_ID = 'breadcrumbs.chevron_left_stroke_icon';
+
+const getGardenId = (rtl: boolean) => (rtl ? RTL_COMPONENT_ID : LTR_COMPONENT_ID);
+
+const ChevronStrokeIcon = (props: SVGAttributes<SVGElement>) => {
+  const { rtl } = useContext(ThemeContext);
+
+  return rtl ? <ChevronLeftStrokeIcon {...props} /> : <ChevronRightStrokeIcon {...props} />;
+};
+
+/**
+ * Accepts all `<svg>` props
+ */
+export const StyledChevronIcon = styled(ChevronStrokeIcon).attrs(props => ({
+  'data-garden-id': `${getGardenId(props.theme.rtl)}`,
+  'data-garden-version': PACKAGE_VERSION,
+  'aria-hidden': 'true'
+}))`
+  display: inline-block;
+  position: relative;
+  top: 1px;
```

Comment:
This pixel pushing isn't great for maintainability so I'm going to look into using flexbox to center the breadcrumb separator.